src/queries/multi-match-query.js

```diff
@@ -3,12 +3,14 @@
  *
  * @param  {Array}  fields  The field names to query over.
  * @param  {String} query   The query string.
+ * @param  {String} type    The type of multi_match query (optional).
```

Comment:
Nit: follow the jsdoc convention for optional params http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values.

```
@param {String} [type='best_fields'] The type of multi_match query.
```